aspnetcore/fundamentals/servers/index.md

```diff
@@ -153,6 +153,9 @@ When launching an app from a command prompt in the project's folder, [dotnet run
   * Public-facing edge server connections use HTTP/2, but the reverse proxy connection to Kestrel uses HTTP/1.1.
   * Target framework: Not applicable to IIS out-of-process deployments.
 
+> [!NOTE]
```

Comment:
Is there a better way to add footnotes in markdown files? Ideally, it should be an asterisk in the section above that links to this note.

Reply:
This looks good to me

Reply:
Use `&dagger;` (single dagger) and `&Dagger;` (double dagger). I'm trying to sneak more daggers 🔪 into the docs.